.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.4.0
+
+    working_directory: ~/art-dashboard
+
+    steps:
+      - checkout
+      - run:
+          name: install dependencies
+          command: sudo yarn install
+
+      # run tests!
+      - run:
+          name: run tests
+          command: yarn test
+
+      # - run:
+      #     name: run coverage
+      #     command: sudo yarn coverage
+
+      # - run:
+      #     name: report coverage
+      #     command: sudo yarn coveralls
```

Comment:
You can add a line at the end here and git will ignore that extra line.

Reply:
Fixed, Thanks.